tests/test_repository_tool.py

```diff
@@ -10,25 +9,22 @@
 from taf.utils import to_tuf_datetime_format
 
 
-def test_check_no_key_inserted_for_targets_should_raise_error(taf_happy_path):
-  with pytest.raises(oll_sc.exceptions.SmartCardNotPresentError):
-    taf_happy_path.is_valid_metadata_yubikey('targets', (1,), '123456')
+def test_check_no_key_inserted_for_targets_should_raise_error(taf_happy_path, targets_yk):
+  targets_yk.insert()
```

Comment:
We are keeping inserted yubikey in a global variable and this ensures that there is no inserted yubikeys. I couldn't just call `targets_yk.remove()` because that method removes yubikey if same instance is already inserted.